docker/README.md

```diff
@@ -117,9 +115,9 @@ $ docker run --rm \
   -ti \
   -p 2222:22 \
   -e DISPLAY=$DISPLAY \
-  -v $(pwd -P)/id_rsa:/home/autosubmit/.ssh/id_rsa \
-  -v $(pwd -P)/id_rsa.pub:/home/autosubmit/.ssh/id_rsa.pub \
-  -v $(pwd -P)/authorized_keys:/home/autosubmit/.ssh/authorized_keys \
+  -v ~/.ssh/id_rsa:/home/autosubmit/.ssh/id_rsa \
+  -v ~/.ssh/id_rsa.pub:/home/autosubmit/.ssh/id_rsa.pub \
+  -v ~/.ssh/authorized_keys:/home/autosubmit/.ssh/authorized_keys \
```

Comment:
I think we shouldn't use our own keys.

Above, in this same docs, there's a line like this `docker run --rm -it --entrypoint /keygen.sh linuxserver/openssh-server`.

That line calls the linuxserver container utility tool to produce SSH keys. They are then stored locally (if I recall correctly).

These docs were showing the user how to produce random keys and use from the local directory (thus `$(pwd -P)`) for testing AS.

Not every user will want to map their keys inside container (I am in this group, for instance :grimacing:), so we either keep the previous version, or we change the docs where we show users how to create keys, and alert them about risks of binding their keys inside the container -- I find it easier to avoid a security risk like this whenever we can...